Makefile

```diff
@@ -250,7 +250,7 @@ endif
 test-sim-nondeterminism:
 	@echo "Running non-determinism test..."
 	@cd ${CURRENT_DIR}/simapp && go test -mod=readonly -run TestAppStateDeterminism -Enabled=true \
-		-NumBlocks=100 -BlockSize=200 -Commit=true -Period=0 -v -timeout 24h
+		-NumBlocks=100 -BlockSize=200 -Commit=true -Period=0 -v -timeout 24h -Seed=6425865534782297734
```

Comment:
Can we get a comment on the reason of hard-coding the seed?
This makes running that sim not more random anymore right?

Reply:
yeah we should not hard code a seed

Reply:
sorry, I was testing and forgot to remove it!